datapipelines/src/main/kotlin/io/customer/sdk/DataPipelineInstance.kt

```diff
@@ -153,6 +153,57 @@ abstract class DataPipelineInstance : CustomerIOInstance {
         track(name, properties, JsonAnySerializer.serializersModule.serializer())
     }
 
+    /**
+     * The screen methods represents screen views in your mobile apps
+     *
+     * @param name A name for the screen.
+     * @param properties Additional details about the screen.
+     * @see [Learn more](https://customer.io/docs/cdp/sources/source-spec/screen-spec/)
+     */
+    @JvmOverloads
+    fun screen(name: String, properties: JsonObject = emptyJsonObject) {
+        screen(name, properties, JsonAnySerializer.serializersModule.serializer())
+    }
+
+    /**
+     * The screen methods represents screen views in your mobile apps
+     *
+     * @param name A name for the screen.
+     * @param properties Additional details about the screen in Map <String, Any> format.
+     * @see [Learn more](https://customer.io/docs/cdp/sources/source-spec/screen-spec/)
+     */
+    fun screen(name: String, properties: CustomAttributes) {
```

Comment:
I think we decided to use `title` here instead of `name` to match iOS and analytics?

```suggestion
    fun screen(title: String, properties: CustomAttributes) {
```

Reply:
hmm, I tried to follow https://customer.io/docs/cdp/sources/source-spec/screen-spec/ but seems like there is a discrepancy.